iot/manager/manager.js

```diff
@@ -120,34 +126,42 @@ const createRegistry = async (
 ) => {
   // [START iot_create_registry]
   // Client retrieved in callback
-  // getClient(serviceAccountJson, function(client) {...});
   // const cloudRegion = 'us-central1';
   // const projectId = 'adjective-noun-123';
   // const registryId = 'my-registry';
   // function errCb = lookupRegistry; // Lookup registry if already exists.
-  const parentName = `projects/${projectId}/locations/${cloudRegion}`;
-  const pubsubTopic = `projects/${projectId}/topics/${pubsubTopicId}`;
+  const iot = require('@google-cloud/iot');
 
+  // Lookup the pubsub topc
+  const topicPath = `projects/${projectId}/topics/${pubsubTopicId}`;
+
+  const iotClient = new iot.v1.DeviceManagerClient({
+    // optional auth parameters.
+  });
+
+  const newParent = iotClient.locationPath(projectId, cloudRegion);
+  const deviceRegistry = {
+    eventNotificationConfigs: [
+      {
+        pubsubTopicName: topicPath,
+      },
+    ],
+    id: registryId,
+  };
   const request = {
-    parent: parentName,
-    resource: {
-      eventNotificationConfigs: [
-        {
-          pubsubTopicName: pubsubTopic,
-        },
-      ],
-      id: registryId,
-    },
+    parent: newParent,
+    deviceRegistry: deviceRegistry,
   };
 
   try {
-    const {data} = await client.projects.locations.registries.create(request);
+    const responses = await iotClient.createDeviceRegistry(request);
+    const response = responses[0];
 
     console.log('Successfully created registry');
-    console.log(data);
+    console.log(response);
   } catch (err) {
     console.log('Could not create registry');
```

Comment:
Can we make this (and other error logging) something like:

`console.error('Could not create registry: ', err)`